packages/tools/kadena-cli/src/account/utils/createAndTransferFunds.ts

```diff
@@ -78,15 +78,16 @@ export async function createAndTransferFund({
       throw new Error('Transaction is not signed');
     }
 
-    const { submit, listen } = createClient(
+    const { submit, local } = createClient(
       `${networkConfig.networkHost}/chainweb/0.0/${networkConfig.networkId}/chain/${chainId}/pact`,
     );
 
-    const transactionDescriptor = await submit(signedTx);
+    // Validate the transaction locally before sending it to the network
+    await local(signedTx);
```

Comment:
does `local()` throw if something is wrong? or should the result be actually validated?

Reply:
yes, it does throws error but for some use cases needs to be validated. so added that check as well.